orchagent/portsorch.cpp

```diff
@@ -1873,6 +1874,22 @@ void PortsOrch::initPortSupportedSpeeds(const std::string& alias, sai_object_id_
     m_portStateTable.set(alias, v);
 }
 
+bool PortsOrch::isSpeedSupported(const std::string& alias, sai_object_id_t port_id, sai_uint32_t speed)
+{
+    // This method will return false if we get a list of supported speeds and the requested speed
+    // is not supported
+    // Otherwise the method will return true (even if did not receive list of supported speed)
+
```

Comment:
Since we removed `initPortSupportedSpeeds` in this function, just want to make sure `initPortSupportedSpeeds` was called always before calling this function.